tests/l2/qat/qatlib_build.yaml

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: intel-qat-workload
+  namespace: intel-qat
+spec: {}
+---
+apiVersion: build.openshift.io/v1
+kind: BuildConfig
+metadata:
+  name: intel-qat-workload
+  namespace: intel-qat
+spec:
+  triggers:
+    - type: "ConfigChange"
+    - type: "ImageChange"
+  runPolicy: "Serial"
+  source:
+    type: Dockerfile
+    dockerfile: |
+        
+        FROM registry.access.redhat.com/ubi8/ubi AS builder
+        ARG QATLIB_VERSION="23.02.0"
```

Comment:
```suggestion
        ARG QATLIB_VERSION="23.08.0"
```

Reply:
it's not there yet but it would be good to move to it if it happens before your release date

Reply:
23.08.0 has released, updated it